doc/introduction/circuits.rst

```diff
@@ -14,18 +14,14 @@ Quantum Circuits
     :target: javascript:void(0);
 
 
-In PennyLane, variational quantum circuits are represented as *quantum node* objects. A quantum node
-is a combination of a quantum function that composes the circuit,
-and a device that runs the computation. One can conveniently create quantum nodes using
-the quantum node decorator.
+In PennyLane, quantum circuits are represented as *quantum node* objects. A quantum node is used to 
+declare the quantum circuit, and also ties the computation to a specific device that executes it. 
+Quantum nodes can be easily created by using the :ref:`qnode <intro_vcirc_decorator>` decorator.
```

Comment:
Just out of curiosity, is this change more or less to remove mention of 'quantum function' for clarity?

Reply:
Yep.